### PR TITLE
feat(xod-client): add tooltips for unlabeled fields/buttons in ProjectBrowser and Inspector

### LINKED ISSUE #725: As a user I want Inspector and Project Browser to have tooltips
### Rationale

As of now I couldn't find tooltips anywhere. They would help new and experienced users alike by sparing them a trip to the documentation.

I would propose tooltips for the following:

- *Label a node* (textbox with green flag) and *View documentation* button. Pretty self explanatory but I personally still like to know exactly what a button does before pressing it. Even more so a button that does an action (opening a new browser window) outside of the application. So preferably the tooltip would for the *View documentation* would read something like "Open documentation for this node **in your browser**".  (See [here](https://i.imgur.com/Ds8IW5v.png))
- *Add node* and *View node documentation* button ([here](https://i.imgur.com/ZqO325b.png)) 
- *Delete patch* and *Rename patch* in "My Patches"

### Acceptance criteria

Tooltips are shown for:

- [x] Inspector → Node Label field
- [x] Inspector → View Documentation button
- [x] Project Browser → Add Node field
- [x] Project Browser → View Documentation button
- [x] Project Browser → Delete Patch button
- [x] Project Browser → Rename Patch button

### How to implement

Tooltips can be added to HTML by using ``title`` attribute. Node inputs/outputs can get a tooltip by using a ``<title></title>`` child in the SVG.

### PR DESCRIPTION
**Pull request by evgenykochetkov**
Closes #725

### CODE REVIEW

File: packages/xod-client/src/editor/components/NodeInspector.jsx [hunk 1]
```diff
@@ -87,6 +87,7 @@ const NodeInspector = ({ node, onPropUpdate }) => {
         target="_blank"
         rel="noopener noreferrer"
         className="nodeHelp"
+        title="Open documentation for this node in your browser"
```
Comment (nkrkv) [inline packages/xod-client/src/editor/components/NodeInspector.jsx]: Maybe shorten to “Open documentation in web browser”?

File: packages/xod-client/src/editor/components/NodeInspector.jsx [hunk 2]
```diff
@@ -110,6 +111,7 @@ const NodeInspector = ({ node, onPropUpdate }) => {
         kind={NODE_PROPERTY_KIND.PROP}
         keyName={NODE_PROPERTY_KEY.LABEL}
         label="Label"
+        title="Label a node"
```
Comment (nkrkv) [inline packages/xod-client/src/editor/components/NodeInspector.jsx]: “Node’s label”?

File: packages/xod-client/src/projectBrowser/containers/ProjectBrowser.jsx [hunk 3]
```diff
@@ -166,6 +169,7 @@ class ProjectBrowser extends React.Component {
         rel="noopener noreferrer"
         className="hover-button"
         key="patch-guide-button"
+        title="Open documentation for this patch in your browser"
```
Comment (nkrkv) [inline packages/xod-client/src/projectBrowser/containers/ProjectBrowser.jsx]: Ditto

File: packages/xod-client/src/projectBrowser/containers/ProjectBrowser.jsx [hunk 4]
```diff
@@ -105,12 +105,14 @@ class ProjectBrowser extends React.Component {
       <Icon
         key="delete"
         name="trash"
+        title="Delete Patch"
```
Comment (nkrkv) [inline packages/xod-client/src/projectBrowser/containers/ProjectBrowser.jsx]: It’s better to be consistent about tooltip case. In other places, we use the regular case, so this should be “Delete patch”.

The main menu (if you recall it) and button labels is another story. They’re traditionally in “Newspaper Case”.

See Atom for example.

File: packages/xod-client/src/projectBrowser/containers/ProjectBrowser.jsx [hunk 5]
```diff
@@ -105,12 +105,14 @@ class ProjectBrowser extends React.Component {
...
         className="hover-button"
         onClick={() => requestDelete(patchPath)}
       />,
       <Icon
         key="rename"
         name="pencil"
+        title="Rename Patch"
```
Comment (nkrkv) [inline packages/xod-client/src/projectBrowser/containers/ProjectBrowser.jsx]: Ditto

File: packages/xod-client/src/projectBrowser/containers/ProjectBrowser.jsx [hunk 6]
```diff
@@ -152,6 +154,7 @@ class ProjectBrowser extends React.Component {
       <Icon
         key="add"
         name="plus-circle"
+        title="Add Node"
```
Comment (nkrkv) [inline packages/xod-client/src/projectBrowser/containers/ProjectBrowser.jsx]: Ditto

**Comment by evgenykochetkov** [comment]
@nkrkv done

**Comment by nkrkv** [comment]
LGTM

**Pull request merged**